container/devel/Dockerfile.openqa.leap

```diff
@@ -0,0 +1,34 @@
+FROM registry.opensuse.org/opensuse/leap:15.6
+
+RUN zypper ar -G https://download.opensuse.org/repositories/devel:/openQA/15.6 devel-openqa \
+        && zypper ar -G https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.6/15.6/ devel-openqa-leap \
+        && zypper ar -G https://download.opensuse.org/repositories/SUSE:/CA/15.6/SUSE:CA.repo \
+        && zypper -n ref \
+    && true
+
+RUN zypper -n install ca-certificates-suse
+
+# openqa
+RUN zypper -n ref && zypper -n install \
+        os-autoinst-devel openQA-devel \
+        expect \
+        nodejs \
+        openssh-clients \
+    && true
+
+# scripts
+RUN zypper -n install \
+        os-autoinst-scripts-deps \
+        os-autoinst-scripts-deps-devel \
+        python3-pytest \
+        python3-virtualenv \
+    && true
+
+# Run custom scripts
+# Put your scripts under ./scripts. Filename should start with custom
+# to be ignored by git
+RUN mkdir /tmp/userscripts
+COPY etc/runuserscripts.sh /tmp/
+COPY scripts/*.sh /tmp/userscripts
+RUN /tmp/runuserscripts.sh
```

Comment:
Why not just mount in a volume with `-v` instead of copying?

Reply:
done